x-pack/test_serverless/api_integration/test_suites/common/alerting/summary_actions.ts

```diff
@@ -63,32 +64,46 @@ export default function ({ getService }: FtrProviderContext) {
       'kibana.version',
       'kibana.alert.consecutive_matches',
     ];
+    const svlCommonApi = getService('svlCommonApi');
+    const svlUserManager = getService('svlUserManager');
+    const supertestWithoutAuth = getService('supertestWithoutAuth');
+    let roleAuthc: RoleCredentials;
+    let internalReqHeader: InternalRequestHeader;
+
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+      internalReqHeader = svlCommonApi.getInternalRequestHeader();
+    });
 
     afterEach(async () => {
       await supertest
         .delete(`/api/actions/connector/${connectorId}`)
-        .set('kbn-xsrf', 'foo')
-        .set('x-elastic-internal-origin', 'foo')
+        .set(internalReqHeader)
+        .set(roleAuthc.apiKeyHeader)
         .expect(204);
       await supertest
```

Comment:
should this be `supertestWithoutAuth`?

Reply:
I'm sure Pierre will laugh at me on this one lol
Thanks very much

https://github.com/elastic/kibana/pull/183381/commits/c82ccd35cca7bd8cc5c9988c139f3e666cf96ff6